appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -372,6 +372,13 @@
   @Description("Error message if the gallery id does not exist")
   String galleryIdNotExist();
 
+  // Deprecating FusionTables
+
+  @DefaultMessage("The FusiontablesControl has been deprecated and will " +
```

Comment:
Why not reuse the existing warning text? If we are going to keep different warning text, consider prefixing this with "This project uses FusiontablescControl. " The other text also includes a link to "Learn More", which this one doesn't have.